odml/addons.py

```diff
@@ -0,0 +1,23 @@
+import odml.tools.xmlparser as X
+import odml.tools.jsonparser as Y 
+
+def load(filename, backend="xml"):
+	if backend == "xml":
+		return X.load(filename)
+	#elif backend == "json":
+	#	return Y.JSONReader().fromFile(open(filename))
+		
+
+def save(obj, path, backend="xml"):
+	if backend == "xml":
+		X.XMLWriter(obj).write_file(path)
+	elif backend == "json":
+		Y.JSONWriter(obj).write_file(path)
+
```

Comment:
These methods are shorter forms of the original methods, just to simplify the load/save. They just call the xmlparser/jsonparser inside odml/tools. The actual work done is by the methods inside them.
So if there exists an error, the method calls inside them which does the actual processing, would lead to the error, in my opinion.